libcxx/include/__bit/countr.h

```diff
@@ -40,6 +43,9 @@ _LIBCPP_NODISCARD _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX14 int __coun
   if (__t == 0)
     return numeric_limits<_Tp>::digits;
 
+#if __has_builtin(__builtin_ctzg)
+  return __builtin_ctzg(__t);
+#else  // __has_builtin(__builtin_ctzg)
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html#index-_005f_005fbuiltin_005fctzg

```suggestion
#if __has_builtin(__builtin_ctzg, numeric_limits<_Tp>::digits)
  return __builtin_ctzg(__t);
#else  // __has_builtin(__builtin_ctzg)
  if (__t == 0)
    return numeric_limits<_Tp>::digits;
```

Reply:
my bad on this suggestion!